LibertyDSNP/example-client
# Feature/show others profiles/177348273

Purpose
---------------
Feature: As a user, I want to see other users' profiles.
[https://www.pivotaltracker.com/story/show/177348273](url)

Solution
---------------
when you click on another user's profile picture from the feed, I want the profile component to load in that user's profile information
acceptance criteria:

switches from your profile information to the selected user's profile information
back arrow navigates back to your profile
does NOT show other user's corresponding feed posts
does NOT include follow/unfollow functionality

Change summary
---------------
* Show other users' profiles on click of their avatar or name/handle by adding a reducer to the user redux store
* make profile block stick to top
* remove edit button if not your profile
* create back button to go back to your profile
* create header in nav to show who's profile you're seeing
* write tests


Steps to Verify
----------------
1. login
2. click on other users' profiles
3. logout
4. log back in and see that it shows your profile
5. click on another users' profile
6. press the back button to see your profile come back


Additional details / screenshot
----------------
https://user-images.githubusercontent.com/43625033/131540560-ddd31d1e-5dea-4f0f-a8bb-b03b52daf47d.mov
